.github/workflows/storybook.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and Deploy Storybook
+on:
+  push
+  # Trigger the action only when files change in the folders defined here
```

Comment:
I still believe the docs should be deployed only upon release — unless we also want a WIP docs version (corresponding to the `main` branch) which might be useful as well. As a developer, I want to see the docs of the _latest version released_ so I know what I can use. But again, this can be discussed at any point in the future.